aws_lambda_powertools/logging/formatter.py

```diff
@@ -77,6 +81,7 @@ def __init__(
         log_record_order: List[str] | None = None,
         utc: bool = False,
         use_rfc3339: bool = False,
+        include_stacktrace: bool = None,
```

Comment:
I'm thinking that we could improve the naming here (naming is hard!).

If I understand correctly the feature, we already include the stack trace in the log entry, but now we are serializing it.

If this is the case, should we maybe make it more explicit by renaming the option to `serialize_stacktrace`? The current name makes me think that if this is `False` I won't have any stack trace in my logs, which is not the case.

Also, should the name be plural `serialize_stacktraces` or singular `serialize_stacktrace`?

Reply:
This was a good suggestion. I updated to serialize_stacktrace. I believe singular is correct for this scenario.